autonomy/analyse/abci/app_spec.py

```diff
@@ -430,10 +435,14 @@ def check_all(cls, packages_dir: Path) -> None:
         did_not_match = []
         fsm_specifications = sorted(
             [
-                *packages_dir.glob("**/fsm_specification.yaml"),
-                *packages_dir.glob("**/fsm_specification_composition.yaml"),
```

Comment:
There's no reason for two formats. We originally had it when the design was less progressed.